docs/src/main/asciidoc/security-overview.adoc

```diff
@@ -50,7 +50,7 @@ Guidance for testing Quarkus Security features and ensuring that your Quarkus ap
 === Cross-origin resource sharing
 
 To make your Quarkus application accessible to another application running on a different domain, you need to configure cross-origin resource sharing (CORS).
-For more information about the CORS filter that Quarkus provides, see the Quarkus xref:http-reference.adoc#cors-filter[CORS filter] section of the "HTTP reference" guide.
+For more information about the CORS filter that Quarkus provides, see the xref:security-cors.adoc#cors-filter[CORS filter] section of the Quarkus "Cross-origin resource sharing" guide.
```

Comment:
There is no "security-cors.adoc" in 3.2 branch

Reply:
approved for other changes... assume this one change should remain as it was?

Reply:
Oops. Fixed. 

@rsvoboda & @ebullient - Thanks for reviewing.